### DISCUSSION TITLE
颜色和分栏没有生效

### DISCUSSION THREAD

**Discussion by karellincoln** [MadMaxChow/VLOOK#122] [Q&A]
您好请问下面是怎么什么问题导致的
![image](https://github.com/MadMaxChow/VLOOK/assets/12014068/ba02f0fd-0639-4273-9184-cfe6e3ea500d)

**Comment by karellincoln** [comment]
![image](https://github.com/MadMaxChow/VLOOK/assets/12014068/1d49e5db-8a78-43ea-9333-c8759111b0c0)
当前最新的版本。
![image](https://github.com/MadMaxChow/VLOOK/assets/12014068/d0f45ad7-2365-4314-a4e1-089a99d81cd5)
配置也是按照您的文档修改了

**Comment by MadMaxChow** [comment]
1、引用块的颜色标识，在编辑时是提示性的，具体渲染是在导出为HTML后再进行。
2、分栏目前看是三栏，但应该有些特殊格式原因被挤到第2行去了，你方便的话把对应的MD文件、使用的主题文件、以及macOS的版本提供一下。
